src/views/camp/camp.jsx

```diff
@@ -0,0 +1,137 @@
+var React = require('react');
+var injectIntl = require('react-intl').injectIntl;
+var FormattedMessage = require('react-intl').FormattedMessage;
+var render = require('../../lib/render.jsx');
+var TitleBanner = require('../../components/title-banner/title-banner.jsx');
+var FlexRow = require('../../components/flex-row/flex-row.jsx');
+
+var Page = require('../../components/page/www/page.jsx');
+
+require('./camp.scss');
+
+var Camp = injectIntl(React.createClass({
+    type: 'Camp',
+    render: function () {
+        return (
+            <div>
+                <TitleBanner className="masthead mod-blue-bg">
+                    <img src="/images/camp/ocean-top.svg" alt="Top" className="topImg"/>
+                    <div className="inner">
+                        <div className="title-content">
+                            <h1 className="title-banner-h1">
+                                <FormattedMessage id='camp.title' />
+                            </h1>
+                            <h4 className="intro title-banner-p">
+                                <FormattedMessage id='camp.dates' />
+                            </h4>
+                            <img src="/images/camp/bubbles.svg" alt="Bubbles" className="bubbles"/>
+                        </div>
+                    </div>
+                </TitleBanner>
+                <div className="gradient1">
+                    <section id="particpate">
+                        <div className="inner">
+                            <center>
+                                <h2><FormattedMessage id='camp.welcome' /></h2>
+                                <p id="intro">
+                                    <h2><FormattedMessage id='camp.welcomeIntro' /></h2>
+                                </p>
+                            </center>
+                            <center>
+                                <img src="/images/camp/fish-divider.svg" className="fishDivider" />
+                            </center>
+                            <h2>
+                                <center><FormattedMessage id='camp.part1Dates' /></center>
+                            </h2>
+                            <center>
+                            <FlexRow className="sidebar-row">
+                                <div className="body-copy column">
+                                    <h3><FormattedMessage id='camp.detailsTitle' /></h3>
+                                    <p>
+                                        <FormattedMessage id='camp.part1Details' />
+                                    </p>
+                                </div>
+                                <div className="sidebar column">
+                                    <h3><FormattedMessage id='camp.particpateTitle' /></h3>
+                                    <p>
+                                        <FormattedMessage id='camp.part1Particpate' />
```

Comment:
I think this needs to be a `FormattedHTMLMessage` component because it contains a link in it